src/config/config-file-utilities.lisp

```diff
@@ -0,0 +1,88 @@
+; notional code to read configuration files and generate
```

Comment:
@svspire Needs to be hooked up into an ASDF definition.  I provided code for you to use in <https://github.com/emotiq/emotiq/tree/config-mte/src/config>